sql/moz-fx-data-shared-prod/fivetran_costs_derived/daily_connector_costs_v1/query.sql

```diff
@@ -0,0 +1,67 @@
+WITH incremental_mar AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.fivetran_costs_derived.incremental_mar_v1`
+),
+destinations AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.fivetran_costs_derived.destinations_v1`
+),
+monthly_costs AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.fivetran_costs_derived.monthly_costs_v1`
+),
+monthly_costs_per_mar AS (
+  SELECT
+    destination_id,
+    measured_month,
+    monthly_costs.dollars_spent / NULLIF(SUM(incremental_mar.active_rows), 0) AS cost_per_mar
+  FROM
+    incremental_mar
+  LEFT JOIN
+    monthly_costs
+  USING
+    (measured_month, destination_id)
+  WHERE
+    incremental_mar.billing_info = "paid"
+  GROUP BY
+    measured_month,
+    destination_id,
+    dollars_spent
```

Comment:
ah thx .. you told me before but I forgot again ...

Reply:
on the second point ... do you also mean in the group by? in this case it makes sense since the dollars_spent are inside a calculation and not a column of the result. But if it is a result column would you still group by the qualified column it came from?

Reply:
For consistency I personally prefer to qualify columns everywhere (except those used in `USING (...)` clauses), even in `GROUP BY`, but using the result column names is also perfectly valid.

I actually usually sidestep that issue entirely by using numerical result column references in `GROUP BY`.  That doesn't help when you need to group by something that isn't a result column (e.g. `dollars_spent` in this case), but I also tend to avoid grouping by non-result columns because I think doing that makes queries harder to read/reason about (e.g. you could pre-calculate the monthly paid rows in another CTE, making the `monthly_costs_per_mar` CTE simpler).